packages/auth/src/tests/signature.ts

```diff
@@ -0,0 +1,56 @@
+import crypto from 'crypto'
+import { importJWK } from 'jose'
+import { KID_ORIGIN, KID_PATH } from '../grant/routes.test'
+
+export const SIGNATURE_METHOD = 'GET'
+export const SIGNATURE_TARGET_URI = '/test'
+
+export const TEST_CLIENT_KEY = {
+  proof: 'httpsig',
+  jwk: {
+    kid: KID_ORIGIN + KID_PATH,
+    x: 'hin88zzQxp79OOqIFNCME26wMiz0yqjzgkcBe0MW8pE',
+    kty: 'OKP',
+    alg: 'EdDSA',
+    crv: 'Ed25519',
+    key_ops: ['sign', 'verify'],
+    use: 'sig'
+  }
+}
+
+export async function generateSigHeaders(
+  url: string,
+  method: string,
+  body?: unknown
+): Promise<{ sigInput: string; signature: string; contentDigest?: string }> {
+  const privateKey = {
+    ...TEST_CLIENT_KEY.jwk,
+    d: 'v6gr9N9Nf3AUyuTgU5pk7gyNULQnzNJCBNMPp5OkiqA'
```

Comment:
Why is this hard-coded?

Reply:
Originally was for bootstrapping the tests faster. Will change.